01/SwitchStatementExample.c

```diff
@@ -0,0 +1,21 @@
+#include <stdio.h>
+
+int main()
+{
+	int x = 3;
+	switch (x)
+	{
+		case 0:

```

Comment:
case statement needs to be at the level of the closing brace
eg:
switch (x) {
case 0:
    printf("Case 0\n");
}